src/plugins/overlay/reports/a11y-1-en.hbs

```diff
@@ -0,0 +1,77 @@
+---
+{
+	"title": "Accessibility assessment #3 - Overlay",
+	"language": "en",
+	"description": "Partial. Evaluation of new Success Criteria at Level A and Level AA of WCAG 2.1 and WCAG 2.2",
+	"tag": "overlay",
+	"parentdir": "overlay",
+	"parentPage": "Overlay",
+	"parentPageURL": "overlay",
+	"altLangPage": "a11y-1-fr.html",
+	"dateModified": "2022-08-19"
+}
+---
+<div id="ds-a11y" data-wb-jsonmanager='{ "url": "a11y-1.json", "name": "a11y" }'></div>
+
+<dl class="dl-horizontal">
+	<dt>Evaluation Report Identifier</dt>
+	<dd data-json-replace="#[a11y]/id">Not Available</dd>
+	<dt>Type</dt>
+	<dd>Accessibility assessment</dd>
+	<dt>Asserted by:</dt>
+	<dd data-json-replace="#[a11y]/assertedBy">Not Available</dd>
+	<dt>Date:</dt>
+	<dd data-json-replace="#[a11y]/date">Not Available</dd>
+	<dt>Page:</dt>
+	<dd><a data-wb-json='[
+
+			{
+				"url": "#[a11y]/subjectUrl",
+				"type": "attr",
+				"attr": "href"
+			},
+			{
+				"url": "#[a11y]/subjectUrl",
+				"type": "replace"
+			}
+
+		]' href="#">Not Defined</a></dd>
+	<dt>Subject</dt>
+	<dd data-json-replace="#[a11y]/assessment">Not Defined</dd>
+	<dt>Notes:</dt>
+	<dd data-json-replace="#[a11y]/notes">No notes</dd>
+	<dt>Scores</dt>
```

Comment:
Since there is only one final percentage, should we write "Score" singular without an "s" ate the end?